desktop/release-notes.md

```diff
@@ -23,17 +23,84 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+> [RPM](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.10.0-x86_64.rpm?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){: .button .primary-btn }
+> [Arch (experimental)](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.10.0-x86_64.pkg.tar.zst?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){: .button .primary-btn }
+
+### Security
+
+Update Go runtime to 1.18.3 to address:
+
+- [CVE-2022-30634](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-30634)
+- [CVE-2022-30629](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-30629)
+- [CVE-2022-30580](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-30580)
+- [CVE-2022-29804](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-29804)
+
+### Upgrades
+- [Docker Compose v2.6.1](https://github.com/docker/compose/releases/tag/v2.6.1)
+- [Kubernetes v1.41.1](https://github.com/kubernetes/kubernetes/releases/tag/v1.41.1)
+- [cri-dockerd to v0.2.1](https://github.com/Mirantis/cri-dockerd/releases/tag/v0.2.1)
+- [CNI plugins to v1.1.1](https://github.com/containernetworking/releases/tag/v1.1.1)
+- [Docker Engine to v20.10.17](https://docs.docker.com/engine/release-notes/#201017)
+- [containerd to v1.6.6](https://github.com/containerd/containerd/releases/tag/v1.6.6)
+- [runc to v1.1.2](https://github.com/opencontainers/runc/releases/tag/v1.1.2)
+- [Go 1.18.3](https://github.com/golang/go/releases/tag/go1.18.3)
+
+### New
+
+- Expanded / collapsed state of the Compose apps is now persisted.
+- External URLs can refer to extension details in Docker Desktop Marketplace using links like `docker-desktop://extensions/marketplace?extensionId=docker/logs-explorer-extension`.
+- Containers overview: added bulk actions for start/pause/stop selected containers.
+- Containers overview: pause/restart actions for compose.
+- Implemented changes on the logs search functionality and added the possibility of clearing the terminal.
+- `docker extension` CLI commands are available with Docker Desktop by default.
+- Disabled Homepage while working on a new design. You can provide [feedback here](https://docs.google.com/forms/d/e/1FAIpQLSfYueBkJHdgxqsWcQn4VzBn2swu4u_rMQRIMa8LExYb_72mmQ/viewform?entry.1237514594=4.10).
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Fixed a bug where a Docker extension fails to load if its backend container(s) are stopped. Fixes [docker/extensions-sdk#16](https://github.com/docker/extensions-sdk/issues/162).
+- Fixed a bug where license agreement is not showing up, thus silently blocking Docker Desktop startup.
+- Fixed the displayed image and tag for unpublished extensions to actually display the ones from the installed unpublished extension.
+- Fixed the duplicate footer on the Support screen.
```

Comment:
Don’t we have any for-win/for-mac links for these?

Reply:
Dunno, I'll ping author accordingly